src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -17,18 +17,27 @@ import {
   MdAutocomplete,
   getMdAutocompleteMissingPanelError,
 } from './index';
+import {
+  ENTER,
+  DOWN_ARROW,
+  SPACE,
+  UP_ARROW,
+  ESCAPE,
+  Directionality,
+  Direction,
+  RxChain,
+  map,
+  startWith
+} from '@angular/cdk';
```

Comment:
I see what you mean but I think this is more like a separate thing we need to revisit later.

It goes kind of back to the initial approach with the secondary entry-point and requires us to rework the packaging (big changes)

For now changing the imports shouldn‘t make it easier to make such a change in the future (we already import frequently from `core.ts` and it‘s not that difficult)